autogpt_platform/frontend/src/components/ui/badge.tsx

```diff
@@ -11,7 +11,7 @@ const badgeVariants = cva(
         default:
           "border-transparent bg-neutral-900 text-neutral-50 shadow dark:bg-neutral-50 dark:text-neutral-900",
         secondary:
-          "border-transparent bg-neutral-100 text-neutral-900 dark:bg-neutral-800 dark:text-neutral-50",
+          "border-transparent bg-neutral-200 text-neutral-900 dark:bg-neutral-800 dark:text-neutral-50",
```

Comment:
use a slightly darker light gray to make it visible on the light gray background the application has in the designs

Reply:
We ended up changing the background color so I reverted this change